VotingApplication/VotingApplication.Web/Scripts/Services/PollService.js

```diff
@@ -115,3 +115,37 @@
         }
     }
 })();
+
+(function () {
+    'use strict';
+
+    angular
+        .module('VoteOn-Create')
+        .factory('PollService', PollService);
+
+    PollService.$inject = ['$http'];
+
+    function PollService($http) {
+
+        var service = {
+            createPoll: createPoll
+        };
+
+        return service;
+
+        function createPoll(poll) {
+            var token = null;
+
+            return $http({
```

Comment:
I know it's a copy/paste from the other service, but we should probably be hiding the fact that it's an http response from the consumer, and just transform the response so it's just the `data` property in the case of a succes.